content/sensu-core/2.0/reference/filters.md

```diff
@@ -210,7 +250,8 @@ checks with a 30 second `interval`.
 This filter evaluates the event timestamp to determine if the event occurred
 between 9 AM and 5 PM UTC on a weekday. Remember that `action` equals to
```

Comment:
I know this isn't your edit, but I was reading this section and noticed this. "Remember that `action` is equal to `allow`" or "Remember that `action` equals `allow`" read a little better than the current line.